File: 1-two-sum/1-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        // Create a hashtable
        // Using Hashtable class
        Hashtable my_hashtable1 = new Hashtable();
        for (int i = 0 ; i < nums.Length ; ++i){
            int remain = target - nums[i];
            if (my_hashtable1.ContainsKey(remain))
                return new int[]{i,(int)my_hashtable1[remain]};
            my_hashtable1[nums[i]]=i;
            
        }
        return new int[]{0,0};
    }
}

File: 1-two-sum/1-two-sum.java
class Solution {
    public static int[] twoSum(int[] nums, int target) {
        for(int i = 0 ; i < nums.length - 1 ; i++)
            for(int j = i + 1 ; j < nums.length ; j++)
            {
                if(nums[i] + nums[j] == target)
                    return new int[]{i  , j };
            }
        return new int[]{-1 , -1};
    }
    
     public static void main(String args[])
    {
        int [] nums = {2 , 7 , 11 , 15};
        int target = 9;
        for(int x : twoSum(nums , target))
            System.out.print(x + " ");
    }
}